src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -679,6 +684,19 @@ public static String formatString(String format, Object... vars) {
     return format;
   }
 
+  public static String safeGetExceptionMessage(String customMessage, Exception ex) {
```

Comment:
whats the reason to use `safe`? Can this be renamed to `generateExceptionMessage`? 
If you are worried about sending null, lets use Optional